src/vivarium/interface/cli.py

```diff
@@ -108,7 +108,7 @@ def run(
 
     start = time()
 
-    results_root = get_output_root(results_directory, model_specification)
+    results_root = get_output_root(results_directory, model_specification, artifact_path)
```

Comment:
Can we add a test for this?

Reply:
Not trivially it seems, without a heavyweight-ish integration test. Created [MIC-3479](https://jira.ihme.washington.edu/browse/MIC-3479).